kratos/containers/data_value_container.cpp

```diff
@@ -19,7 +19,7 @@
 
 
 // Project includes
-#include "includes/define.h"
+#include "containers/variable_component.h"
```

Comment:
One question why did you move this to the cpp?

Reply:
When we had compilation problems due to recursive dependencies